1 test uses 'export IFS' messing up word splitting of $diff_opt.

Upstream issue: https://github.com/magicant/yash/issues/50

--- a/tests/run-test.sh
+++ b/tests/run-test.sh
@@ -305,6 +305,8 @@ testcase() {
             ;;
     esac
 
+    unset -v IFS
+
     # check standard output
     if { <&4; } 2>/dev/null; then
         printf '%% standard output diff:\n'
